src/commcare_cloud/ansible/roles/elasticsearch/templates/config/elasticsearch.yml.j2

```diff
@@ -21,6 +21,7 @@ transport.tcp.port: {{ elasticsearch_tcp_port }}
 http.port: {{ elasticsearch_http_port }}
 
 http.cors.enabled: true
+http.cors.allow-origin: "null"
```

Comment:
Is this a valid setting for v1.7?

Reply:
Appears to be: https://www.elastic.co/guide/en/elasticsearch/reference/1.7/modules-http.html